cypress/plugins/tasks.js

```diff
@@ -84,27 +70,26 @@ const stackConfigTask = (_, config) => {
 const sqlTask = on => {
   on('task', {
     execSql: async sql => {
-      client = new Client(pgConfig)
+      const client = new Client(pgConfig)
       await client.connect()
       const res = await client.query(sql)
       client.end()
-
       return res
     },
-    dropAndSeedDatabase: () => {
-      postgresContainer.stdin.write(
-        `dropdb --if-exists --force ${devDatabase}; psql template1 -c "CREATE DATABASE ${devDatabase} TEMPLATE ${seedDatabase};"\n`,
-      )
-      return Promise.all([
-        new Promise(resolve => exec('tools/bin/mage dev:redisFlush', resolve)),
-        new Promise(resolve => {
-          postgresContainer.stdout.on('data', resolve)
```

Comment:
Another issue with this is that the promises resolves on the first data. That happened to work because the first and only output was from the `psql` command and after that everything was done. However, the `dev:sqlRestore` mage step may do multiple things with output in various stages, and all that has to be awaited before continuing.

Reply:
If you use the mage command then I guess you can just use `.on('close', () => { … })`.